mip/constraint.go

```diff
@@ -0,0 +1,63 @@
+package mip
+
+// Sense defines the constraint operator between the left-hand-side
+// and the right-hand-side.
+type Sense int64
+
+// Sense of a Constraint.
+const (
+	// LessThanOrEqual used to define a less than equal constraint
```

Comment:
```suggestion
	// LessThanOrEqual is used to define a less than or equal constraint
```
Sounds a bit loosely formulated to me otherwise. Maybe I am just off though. We have it in the "type" too though.